Shared/Views/BlurHashView.swift

```diff
@@ -0,0 +1,62 @@
+//
+// Swiftfin is subject to the terms of the Mozilla Public
+// License, v2.0. If a copy of the MPL was not distributed with this
+// file, you can obtain one at https://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) 2022 Jellyfin & Jellyfin Contributors
+//
+
+import SwiftUI
+import UIKit
+
+struct BlurHashView: UIViewRepresentable {
+
+	let blurHash: String
+
+	func makeUIView(context: Context) -> UIBlurHashView {
+		UIBlurHashView(blurHash)
+	}
+
+	func updateUIView(_ uiView: UIBlurHashView, context: Context) {}
+}
+
+class UIBlurHashView: UIView {
+
+	private let imageView: UIImageView
+
+	init(_ blurHash: String) {
+		let imageView = UIImageView()
+		imageView.translatesAutoresizingMaskIntoConstraints = false
+		self.imageView = imageView
+
+		super.init(frame: .zero)
+
+		computeBlurHashImageAsync(blurHash: blurHash) { blurImage in
```

Comment:
Don't we need a weak capture here?

Reply:
Yeah probably, would just be a small optimization.